python/lsst/meas/algorithms/astrometrySourceSelector.py

```diff
@@ -0,0 +1,121 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+
+from lsst.afw import table
```

Comment:
I thought the convention was to import these modules as:
import lsst.afw.table as afwTable